board/app/controllers/comment_controller.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+class CommentController extends AppController {
+
+const WRITE_COMMENT = 'write';
+const WRITE_COMMENT_END = 'write_end';
+const MAX_USER_IN_NEWSFEED = 1;
+
+    public function write() 
+    {
+        $thread_id = Param::get('thread_id');
+        $thread = Thread::get($thread_id);
+        $comment = new Comment();
```

Comment:
I think this declaration is no longer needed, since it is unused plus you're also creating a new instance of Comment at Line26 :bow: